### ISSUE TITLE
Notebook idea: tie in to CMIP6 data

### ISSUE THREAD

**Issue by mnlevy1981** [NCAR/ldcpy#13]
It seems like all the pieces are in place to read in some CMIP6 data, run it through `zfp`, and then create comparison plots.

**Comment by mnlevy1981** [comment]
In the process of setting up an environment on cheyenne (where `intake-esm` can find data), I'll also tackle https://github.com/NCAR/ldcpy/issues/14

**Comment by pinarda** [comment]
@allibco we aren't actually doing this, but two of our notebooks are using CESM-LENS data and I don't think there are any additional steps to using cmip6 data right? It will be in the same format and in the same location (in AWS or glade)?

**Comment by allibco** [comment]
correct.  to use the cmip6 data, we would just access it via intake_esm.
